src/sidebar/websocket.js

```diff
@@ -109,7 +120,20 @@ export default class Socket extends EventEmitter {
 
     /** Close the underlying WebSocket connection */
     this.close = function() {
-      socket.close();
+      // nb. h's ws4py-based WebSocket server will currently use whatever
+      // status code we call `close()` with as the status code in its response.
+      //
+      // If no status code is provided, the browser will send a close frame
```

Comment:
Although it's implied already, I think it could be helpful to amend this clause to read `If no status code is provided in the `close()` call`, as I had to read this a couple of times to disambiguate _who_ was doing the providing here.